source/auth/auth.md

```diff
@@ -1246,20 +1246,24 @@ in the MONGODB-OIDC specification, including sections or blocks that specificall
     performed after SRV record resolution, if applicable. This property is only required for drivers that support the
     [Human Authentication Flow](#human-authentication-flow).
 
-#### Built-in Provider Integrations
+<div id="built-in-provider-integrations">
 
-Drivers MUST support all of the following built-in OIDC providers.
+#### Built-in OIDC Environment Integrations
 
-####### AWS
+Drivers MUST support all of the following built-in OIDC application environment integrations.
 
-The AWS provider is enabled by setting auth mechanism property `PROVIDER_NAME:aws`.
+####### Test
```

Comment:
The [doc](https://docs.google.com/document/d/1KfjHr-114ADX_2GRW6jTwYbRUsMcJZihS3F5xbOQKFc/edit) suggests that there will be a mechanism that uses:

> A file at a path specified by an environment variable

It seems that the "Test" env, and the current env var, will be redundant with this (future?) env. Could we not use that env? This would also make our tests more useful, because they would test an external API rather than internal.

Reply:
The kubernetes case will be the only other one that uses an env variable, this is meant to be run locally or on EVG without being on k8s.